### ISSUE TITLE
Slow training loop with QuantileLoss

### ISSUE THREAD

**Issue by paxcema** [mindsdb/lightwood#320]
* Lightwood version: last staging

Training on `us_health_insurance` is very slow for a dataset of that size. Changing the loss function from `QuantileLoss()` to `MSELoss()` makes the issue dissappear. We should understand why this happens.

**Comment by paxcema** [comment]
Closed by #331
